boards/riscv/it8xxx2_evb/it8xxx2_evb_defconfig

```diff
@@ -16,7 +16,7 @@ CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=32768
 
 CONFIG_XIP=y
 CONFIG_INIT_STACKS=y
-CONFIG_OUTPUT_PRINT_MEMORY_USAGE=n
+CONFIG_OUTPUT_PRINT_MEMORY_USAGE=y
```

Comment:
This already defaults to `y`. There is really no reason for this board defconfig to explicitly configure this, so it should be just removed.

Reply:
Understood, I'll remove it. Thanks for guidance.